go/upgrade/migrations/empty.go

```diff
@@ -0,0 +1,24 @@
+package migrations
+
+const (
+	// EmptyHandler is the name of the upgrade handler that doesn't perform any updates.
+	EmptyHandler = "empty"
```

Comment:
Maybe add a comment that this is useful for upgrades that bump protocol versions, but don't need any other migrations.

Reply:
Done.